### ISSUE TITLE
Bluetooth headphone controls not working?

### ISSUE THREAD

**Issue by rameshdharan** [zsszatmari/MagicKeys#3]
Hi, wasn't sure if the new headphone controls commit applied to bluetooth headsets as well. At any rate I tried the latest version (1.4) and pressing the play/pause button on my bluetooth headset still starts iTunes.

**Comment by smarek** [comment]
I can confirm this issue with 1.4.2 version, is it worth anything to try posted HIDDeviceTest or not?

**Comment by treasurebox** [comment]
Yes, sure, there are many kinds of BT devices, more data is better! Thank You!

**Comment by smarek** [comment]
@treasurebox so I've tried, Mikeys doesn't show anything for my Philips SHB7150FB will trigger iTunes on Play/Pause button, and even if I unload `/System/Library/LaunchAgents/com.apple.rcd.plist` service it shows nothing.

I've tried to integrate https://github.com/jguice/mac-bt-headset-fix/
But the NSKeyDownMask global listener returns nothing.
You can see my effort here: https://github.com/smarek/MagicKeys/commit/1fd2694c66371be839bd6d2e25bc355f634213fb

**Comment by treasurebox** [comment]
Thanks! It might be that it does not work with your BT headphones, but works with @frus 's one. @frus : Does mac-bt-headset-fix still work with Yosemite? (I assume @smarek you've tried with Yosemite now)
